### PR TITLE
test(text-helper): add test for emojifyText, abbreviateNumber

### PR DESCRIPTION
**Pull request by ibotdotout**
#518

### CODE REVIEW

File: __tests__/tests/utilities/text-helper.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { emojifyText, abbreviateNumber } from 'utils';
+
+describe('Text Helper', () => {
+  describe('emojifyText', () => {
+    it('should get correctly display :caffee: with emoji', () => {
```
Comment (andrewda) [inline __tests__/tests/utilities/text-helper.js]: `:caffee:` => `:coffee:` 😉

File: __tests__/tests/utilities/text-helper.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      expected = 'I need more ☕️';
+      result = emojifyText('I need more :coffee:');
```
Comment (andrewda) [inline __tests__/tests/utilities/text-helper.js]: Declare these with `const`.

File: __tests__/tests/utilities/text-helper.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      expect(result).toEqual(expected);
+    });
+  });
+
+  describe('abbreviateNumber', () => {
+    it('should get 1 when give 1', () => {
+      input = 1;
+      expected = 1;
+      result = abbreviateNumber(input);
```
Comment (andrewda) [inline __tests__/tests/utilities/text-helper.js]: Declare these with `const`.

File: __tests__/tests/utilities/text-helper.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      expect(result).toEqual(expected);
+    });
+
+    it('should get 1k when give 1000', () => {
+      input = 1000;
+      expected = '1k';
+      result = abbreviateNumber(input);
```
Comment (andrewda) [inline __tests__/tests/utilities/text-helper.js]: Declare these with `const`.

File: __tests__/tests/utilities/text-helper.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+      expect(result).toEqual(expected);
+    });
+
+    it('should get 1.1k when give 1100', () => {
+      input = 1100;
+      expected = '1.1k';
+      result = abbreviateNumber(input);
+      expect(result).toEqual(expected);
```
Comment (andrewda) [inline __tests__/tests/utilities/text-helper.js]: Declare these with `const`.

File: __tests__/tests/utilities/text-helper.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    });
```
Comment (andrewda) [inline __tests__/tests/utilities/text-helper.js]: I would also add a test for something huge and random like `96234`, which should equal `96.2k` I think.

**Comment by ibotdotout** [comment]
thank you @alejandronanez @andrewda for your reviewed.

I improve
- mock emojifyText instead direct test output, I not sure with sinon.spy that it correct ?
- insert const to variables
- add huge and random 96234 -> 96.2k

**Comment by lex111** [comment]
@ibotdotout coluld you add a new line before `expect`?

File: __tests__/tests/utilities/text-helper.js [hunk 7]
```diff
@@ -0,0 +1,49 @@
+import { emojifyText, abbreviateNumber } from 'utils';
+import sinon from 'sinon';
+import emoji from 'node-emoji';
+
+describe('Text Helper', () => {
+  describe('emojifyText', () => {
+    it('should call correcly with text params', () => {
+      const emojify = sinon.spy(emoji, 'emojify');
+      const input = 'I need more :coffee';
```
Comment (lex111) [inline __tests__/tests/utilities/text-helper.js]: Is there not at the end `:`?

**Comment by ibotdotout** [comment]
@alejandronanez already change to use jest for spy with `jest.spyOn`.

File: __tests__/tests/utilities/text-helper.js [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+      expect(emojify).toBeCalledWith(input);
+
+      emojify.mockReset();
+      emojify.mockRestore();
+    });
+  });
+
+  describe('abbreviateNumber', () => {
+    it('should get 1 when give 1', () => {
```
Comment (andrewda) [inline __tests__/tests/utilities/text-helper.js]: Change this to: `'should return 1 when given 1'`

File: __tests__/tests/utilities/text-helper.js [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+      const input = 1;
+      const expected = 1;
+      const result = abbreviateNumber(input);
+
+      expect(result).toEqual(expected);
+    });
+
+    it('should get 1k when give 1000', () => {
```
Comment (andrewda) [inline __tests__/tests/utilities/text-helper.js]: Apply same pattern as above

File: __tests__/tests/utilities/text-helper.js [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+      const input = 1000;
+      const expected = '1k';
+      const result = abbreviateNumber(input);
+
+      expect(result).toEqual(expected);
+    });
+
+    it('should get 1.1k when give 1100', () => {
```
Comment (andrewda) [inline __tests__/tests/utilities/text-helper.js]: ^

File: __tests__/tests/utilities/text-helper.js [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+      const input = 1100;
+      const expected = '1.1k';
+      const result = abbreviateNumber(input);
+
+      expect(result).toEqual(expected);
+    });
+
+    it('should get 96.2k when give 96234', () => {
```
Comment (andrewda) [inline __tests__/tests/utilities/text-helper.js]: ^

File: __tests__/tests/utilities/text-helper.js [hunk 12]
```diff
@@ -0,0 +1,52 @@
+import { emojifyText, abbreviateNumber } from 'utils';
+import emoji from 'node-emoji';
+
+describe('Text Helper', () => {
+  describe('emojifyText', () => {
+    it('should call correcly with text params', () => {
+      const emojify = jest.spyOn(emoji, 'emojify');
```
Comment (alejandronanez) [inline __tests__/tests/utilities/text-helper.js]: 💯

**Pull request merged**